### PR TITLE
Vaishali/bugfix

### PR DESCRIPTION
**Pull request by VaishaliGovindaraj**
Added changes from Branch  , media query updated

### CODE REVIEW

File: styles/index.css [hunk 1]
```diff
@@ -96,26 +96,69 @@ body {
...
+        height:100%;
+    }
+    
+    .grid_box_parent:hover .grid_box{
+        transform: perspective(1000px) scaleZ(6) rotateZ(40deg); 
+        width: 75%;
+        font-size: 1.5rem;
+        background-color: beige;
+        z-index:100;
+
```
Comment (Svpriyaa2808) [inline styles/index.css:146]: remove white spaces

Comment (VaishaliGovindaraj) [reply styles/index.css:146]: Sure

File: index.html [hunk 2]
```diff
@@ -23,43 +23,59 @@ <h1 class="header__page-title">
...
+                            <svg class="side-nav__icon" version="1.1" xmlns="http://www.w3.org/2000/svg" width="20"
+                            height="20" viewBox="0 0 20 20">
+                            <path
+                                d="M14.090 2.233c-1.14-0.822-2.572-1.233-4.296-1.233-1.311 0-2.418 0.289-3.317 0.868-1.427 0.906-2.185 2.445-2.277 4.615h3.307c0-0.633 0.185-1.24 0.553-1.828 0.369-0.586 0.995-0.879 1.878-0.879 0.898 0 1.517 0.238 1.854 0.713 0.339 0.477 0.508 1.004 0.508 1.582 0 0.504-0.252 0.965-0.557 1.383-0.167 0.244-0.387 0.469-0.661 0.674 0 0-1.793 1.15-2.58 2.074-0.456 0.535-0.497 1.338-0.538 2.488-0.002 0.082 0.029 0.252 0.315 0.252s2.316 0 2.571 0c0.256 0 0.309-0.189 0.312-0.274 0.018-0.418 0.064-0.633 0.141-0.875 0.144-0.457 0.538-0.855 0.979-1.199l0.91-0.627c0.822-0.641 1.477-1.166 1.767-1.578 0.494-0.676 0.842-1.51 0.842-2.5-0.001-1.615-0.571-2.832-1.711-3.656zM9.741 14.924c-1.139-0.035-2.079 0.754-2.115 1.99-0.035 1.234 0.858 2.051 1.998 2.084 1.189 0.035 2.104-0.727 2.141-1.963 0.034-1.236-0.834-2.076-2.024-2.111z">
+                            </path>
+                        </svg>
+                            <span>About US</span>
```
Comment (TobiasKjernell) [inline index.html:78]: small "s" in US

File: styles/index.css [hunk 3]
```diff
@@ -96,26 +96,69 @@ body {
...
+        padding:var(--content-padding);
+    }
+    
+    .workflow{
+        display:grid;
+        grid-template: repeat(3,1fr) / repeat(3,1fr);
+        gap:8px;
+        padding:var(--content-padding);
+    }
+
```
Comment (ranjithavelusamy) [inline styles/index.css:162]: u can remove the space

File: styles/index.css [hunk 4]
```diff
@@ -96,26 +96,69 @@ body {
 .grid_box{
     border: 2px solid #777;
     padding: var(--content-padding);
+    height:100%;
 }
 
-.grid_box:hover{
+/* .grid_box:hover{
```
Comment (Svpriyaa2808) [inline styles/index.css:102]: if its not needed you can remove it.

File: styles/index.css [hunk 5]
```diff
@@ -96,26 +96,69 @@ body {
...
-    gap:8px;
+    display:flex;
+    flex-direction: column;
     padding:var(--content-padding);
 }
+
+
+@media screen and (min-width:768px){
+    
+
```
Comment (andrewkalumba) [inline styles/index.css:133]: spacing

**Comment by TobiasKjernell** [comment]
make sure you have the correct names of the navbar-page-names

**Review by andrewkalumba — APPROVED** [review]
good Vaishali

**Review by TobiasKjernell — APPROVED** [review]

**Review by ranjithavelusamy — APPROVED** [review]
code looks good

**Review by Svpriyaa2808 — APPROVED** [review]
Looks good

**Pull request merged**